Vagrantfile

```diff
@@ -18,12 +18,12 @@ module KatelloDeploy
       :libvirt    => 'http://m0dlx.com/files/foreman/boxes/centos64.box'
     },
     :centos7 => {
-      :box_name   => 'centos7',
+      :box_name   => 'centos7_1',
```

Comment:
I guess I am mostly using boxes.yaml these days anyway, so, I think the forcing people to download it is better than trying to keep the box_name consistent at all times.